.github/workflows/e2e.yml

```
@@ -6,27 +6,33 @@ jobs:
   e2e:
     runs-on: ubuntu-latest
     name: End-to-end tests
+    concurrency:
+      group: react-e2e
+      cancel-in-progress: false
+    env:
+      PLAYWRIGHT_BROWSERS_PATH: ./pw-browsers
     steps:
       - uses: actions/checkout@v3
 
       - name: 💾 Cache Dependencies
         uses: actions/cache@v3
         with:
-          path: ./node_modules
+          path: |
+            ./node_modules
+            ./pw-browsers
           key: ${{ runner.os }}-${{ matrix.node }}-modules-${{ hashFiles('**/yarn.lock') }}
```

```suggestion
          key: ${{ runner.os }}-${{ matrix.node }}-modules-${{ hashFiles('./yarn.lock') }}
```
Let's also get rid of this wildcard, I don't think we care about other lock files at this point in the pipeline.

---

Good point!